server/src/helpers/translation.js

```diff
@@ -47,12 +49,38 @@ const getTranslatedText = async (text, originalLanguage) => {
   }
 }
 
+const readTranslationMockFile = async file => {
+  try {
+    const rawData = await fs.readFile(file)
+    const data = JSON.parse(rawData)
+    return data
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error(error)
+    return {}
+  }
+}
+
+const findTranslationMock = (data, text) => {
+  for (const [, item] of Object.entries(data)) {
+    if (item.title === text || item.content === text) {
+      return item
+    }
+  }
+  return null
+}
```

Comment:
En fait maintenant que je le relis sous cette forme je me rend compte que tu simplement faire

```suggestion
const findTranslationMock = (mocks, text) => {
  return Object.values(mocks).find(({ title, content }) => title === text || content === text)
}
```